api/v08/kogitoserverlessworkflow_types.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v08
 
 import (
+	"encoding/xml"
```

Comment:
Why we need this import?

Reply:
@davidesalerno I think my IDE settings pulled this in...I actually didn't import it...so will get that removed...